src/client/components/Button/Button.js

```diff
@@ -2,12 +2,12 @@ import React from "react";
 import PropTypes from 'prop-types';
 import './Button.styles.css';
 
-const Button = ({ buttonName,backgroundColor}) => {
-  return <button className= "button" type="submit" style= {{backgroundColor}}> {buttonName} </button>;
+const Button = ({buttonName,style}) => {
+  return <button className= "button" type="button" style= {style}> {buttonName} </button>;
 };
 export default Button;
  
 Button.propTypes = {
   buttonName: PropTypes.string.isRequired,
-  backgroundColor:PropTypes.string.isRequired
+  style: PropTypes.string.isRequired
```

Comment:
I think it should be optional. It's an edge case if you want to style the button. The primary styles should reside in the button component itself as i see it. :)

Reply:
ok, should i remove style from here

Reply:
No, you should make it optional. Right now it's required. :)

Reply:
@shruthidasarapu should'nt it also be a object? And i think the default value should just be an empty object.

Reply:
i tried with object but it shows error 
![image](https://user-images.githubusercontent.com/66563360/106238415-29296480-6201-11eb-98af-f83056729965.png)

Reply:
Allright, hm we still have a failing check. Could you see if you can fix that? :)